### DISCUSSION TITLE
Very slow and lagy opening of window

### DISCUSSION THREAD

**Discussion by Oppa-YA** [File-New-Project/EarTrumpet#1417] [Q&A]
Hello, I like your app, but every time when i click on the icon to open main window to change the slider it opens notisable slowly with lags and stutters.
My laptop has i7 + 16GB RAM and latest updated Windows 10 22H2 so that's not an issue. 
Can you please add the ability to disable opening animation? It would make app more usable. Thank you.

**Comment by mahammadismayilov** [comment]
I'am having the same issue and would prefer to disable animations rather than this laggy thing
